README.md

```diff
@@ -56,12 +58,12 @@ $opentok = new OpenTok($apiKey, $apiSecret);
 To create an OpenTok Session, use the `createSession($options)` method of the
```

Comment:
Do we want to include updated instructions in each SDK on setting the new `timeout` parameter?

Reply:
Added to the README and fleshed out all 3 options that can be overriden